Assets/css/styles.css

```diff
@@ -147,24 +140,20 @@ ul.right-bar-ul{
 #abs{
   width: 100%;
 }
-.align-top p{
-  font-size: 16px;
-}
-.align-top a:hover{
-  text-decoration-color: #f72210;
-}
+
 .par-mid{
   font-weight: 400;
 }
 .col-sm-4.col-md-4.mid-col a{
   color: black;
   font-size: 24px;
 }
-.col-sm-4.col-md-4.mid-col a:hover{
-  text-decoration-color: #f72210;
+.red-links p{
+  color: black;
 }
 .red-links p:hover{
   color: #f72210;
+  text-decoration-style: none;
```

Comment:
`none` is not a text-decoration-style. Use `text-decoration: none`. Please address this before merging